tests/functional/persist_docs_tests/test_persist_docs.py

```diff
@@ -1,17 +1,47 @@
-from test.integration.base import DBTIntegrationTest, use_profile
-import os
-
 import json
-
-
-class BasePersistDocsTest(DBTIntegrationTest):
-    @property
-    def schema(self):
-        return "persist_docs_060"
-
-    @property
+import os
+import pytest
+
+from dbt.tests.util import (
+    run_dbt,
+)
+
+from tests.functional.persist_docs_tests.fixtures import (
+    _DOCS__MY_FUN_DOCS,
+    _MODELS__MISSING_COLUMN,
+    _MODELS__NO_DOCS_MODEL,
+    _MODELS__TABLE,
+    _MODELS__VIEW,
+    _PROPERITES__SCHEMA_MISSING_COL,
+    _PROPERTIES__SCHEMA_YML,
+    _SEEDS__SEED,
+)
+
+
+class BasePersistDocsTest:
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
```

Comment:
I don't think this is a fixture, is it? If you want this to run once per class, changing it to this should work:
```
@classmethod
def setup_class(cls):
    ...
```
If you need it for each method, then this should work:
```
def setup_method(self, method):
    ...
```

Reply:
It's not a fixture in the project fixture definition, and I agree with  the setup_class,setup_method methods. However, this is the scheme the team has been using. I'm doing this for current sake of consistency more than anything.